js/src/components/shipping-rate-section/minimum-order-card/changeGroup.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @typedef { import(".~/data/actions").ShippingRate } ShippingRate
+ * @typedef { import("./typedefs").MinimumOrderGroup } MinimumOrderGroup
+ */
+
+/**
+ * Returns new shipping rates value, with updated group.
+ *
+ * Will set `shipingRate.options.free_shipping_threshold` to `newGroup.threshold`
+ * for the shipping rate countries that are listed in `newGroup.countries` (if provided),
+ * and set `threshold` to `undefined` for the countries which were in `oldGroup.countries` (if provided),
+ * but not in the new.
+ *
+ * @param {Array<ShippingRate>} value Shipping rates.
+ * @param {MinimumOrderGroup} [oldGroup] Old minimum order group.
+ * @param {MinimumOrderGroup} [newGroup] New minimum order group.
+ */
+export const changeGroup = ( value, oldGroup, newGroup ) => {
```

Comment:
💅  Since we are doing several refactoring in favor of

```
const changeGroup = () ...

export default changeGroup;

```

I'd suggest to do that way here as well.  Even tho we are not tracking anything in this file we will get more consistency over our codebase

Reply:
💅  What about `changeMinimumOrderGroup` ?

Even tho is located in a folder named like that it could be non descriptive and pretty much generic to use `changeGroup`

Reply:
> Do you mean having default export, or just separating export from statement, to workaround tracking docs generator issue?

I meant this.

Reply:
Personally, I find `export const foo = …` a nice syntax, as it gives you the instant info that the thing being defined will be exported.

I wouldn't employ `const foo = … export foo;` convention just because it's a workaround for jsdoc generator bug. I'd use a workaround where needed (where we track events), and try to fix the bug at its source.